### ISSUE TITLE
lstmtraining killed by signal 9

### ISSUE THREAD

**Issue by Shreeshrii** [tesseract-ocr/tesseract#2016]
Command terminated by signal 9
Command being timed: "lstmtraining --model_output ./layersan/layer --continue_from ./layersan/Devanagari.lstm --append_index 5 --net_spec [Lfx192 O1c1] --traineddata ./layersan/san/san.traineddata --train_listfile ./okfonts.txt --eval_listfile ./fontsamples/san.training_files.txt --debug_interval 0 --max_image_MB 12000 --max_iterations 1210000"
        User time (seconds): 1408.20
        System time (seconds): 16.57
        Percent of CPU this job got: 163%
        Elapsed (wall clock) time (h:mm:ss or m:ss): 14:32.19
        Average shared text size (kbytes): 0
        Average unshared data size (kbytes): 0
        Average stack size (kbytes): 0
        Average total size (kbytes): 0
        Maximum resident set size (kbytes): 16282880
        Average resident set size (kbytes): 0
        Major (requiring I/O) page faults: 13
        Minor (reclaiming a frame) page faults: 582567
        Voluntary context switches: 156871
        Involuntary context switches: 11656
        Swaps: 0
        File system inputs: 38186040
        File system outputs: 881024
        Socket messages sent: 0
        Socket messages received: 0
        Signals delivered: 0
        Page size (bytes): 65536
        Exit status: 0

**Comment by zdenop** [comment]
Did you tried to decrease amounts in `--max_image_MB` and  `--max_iterations`?

**Comment by Shreeshrii** [comment]
max_image_MB default is 6gb. Since that caused the error, I changed it to a higher number. The VM has 16Gb ram and this is the  process running (other than all system related ones).

I have tried restricting/varying the number of openmp threads also.

max_iterations is high because I have been trying this training run since early September. While initially the char error reduces rapidly after getting to less than 10% the progress is slow. It is currently at 1.5%.
